client/jetpack-connect/remote-credentials.js

```diff
@@ -206,7 +188,7 @@ export class OrgCredentialsForm extends Component {
 	render() {
 		return (
 			<MainWrapper>
-				{ this.formHeader() }
+				{ this.renderHeadersText() }
```

Comment:
This is the same situation, let's just inline it.

Reply:
Not exactly I believe as we do not introduce any extra divs nor other elements by separating functionality. IMO it is aligned with having a footer returned from a function. No harm, so I'd leave it as-is. Thanks 👍

Reply:
Appreciate your input Jon! I'll go ahead with merging. We'll have a refactor clean-up before shipping when all pieces will be in place. Expecting some more code to land in this file :)